bedrock/base/templates/500.html

```diff
@@ -2,9 +2,9 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
-{% set_lang_files "mozorg/500" %}
+{% set_lang_files "mozorg/404-update-2020" %}
```

Comment:
Note we still need to keep existing translations around for the 500 page too, like we did for the 404 page.